dp-census-atlas.nomad

```diff
@@ -36,7 +36,7 @@ job "dp-census-atlas" {
       config {
         command = "${NOMAD_TASK_DIR}/start-task"
 
-        args = ["./dp-census-atlas"]
+        args = ["PORT=${PORT:-28100} node build/index.js"]
```

Comment:
```suggestion
        args = ["node", "build/index.js"]
```

Reply:
what does having the commands in seperate array entries mean?

Reply:
the args are not treated as a single string, otherwise you'd need a command (binary) called `node build/index.js` 

rather than a binary `node` with the argument `build/index.js`

Reply:
discussed offline - no it doesn't. Change done.